scipy/stats/_continuous_distns.py

```diff
@@ -16,7 +16,7 @@
 
 from numpy import (where, arange, putmask, ravel, sum, shape,
                    log, sqrt, exp, arctanh, tan, sin, arcsin, arctan,
-                   tanh, cos, cosh, sinh)
+                   tanh, cos, cosh, sinh, where)
```

Comment:
We're trying to move away from `from numpy import everything` to `import numpoy as np; np.something`. Best just use `np.where`.